### ISSUE TITLE
Unresponsive search in Advanced search

### ISSUE THREAD

**Issue by peterwebster** [ukwa/shine#26]
While testing in IE, the Search button in the Advanced search is not responding.

Simple search is OK, so the service is clearly running.

**Comment by peterwebster** [comment]
@anjackson so, here's the bug you found in the middle of your IIPC presentation on Solr

**Comment by peterwebster** [comment]
@anjackson I have had the same issue on machines at STP in meetings

**Comment by anjackson** [comment]
Yes, doesn't work on some IE versions/configurations? Maybe additional request or resources are being blocked? The modal 'busy' indicator does not even come up.

**Comment by anjackson** [comment]
This reminds me of an earlier issue, that I thought I had worked around. The service is classes as being on the intranet, which we have configured to force IE7 mode.  This produces logging info in IE9 like this:

    HTML1202: http://www.webarchive.org.uk/shine/search?query=captcha&tab=results&action=search is running in Compatibility View because 'Display intranet sites in Compatibility View' is checked. 
    search?query=captcha&tab=results&action=search
    HTML1113: Document mode restart from IE7 Standards to IE9 Standards 
    search?query=captcha&tab=results&action=search

But I thought I'd fixed it by doing the 'IE version = EDGE' thing in the header. It would be good to get the browser error log from a failing browser.

**Comment by peterwebster** [comment]
@anjackson do we know whether this is still a live issue ? I can't test it as SHINE seems to be down right now; but if it is, there won't be much hands-on interaction at the grad student event on Friday.

**Comment by peterwebster** [comment]
Also pinging @GilHoggarth re SHINE being down

**Comment by GilHoggarth** [comment]
Seen ping - investigating!

**Comment by GilHoggarth** [comment]
Checking /shine, eventually saw issue reported as 'proxy error'?!
Restarted shine2 successfully (a very easy, quick action) but didn't resolve /shine service being down. Examined jisc1 and jisc2 servers to find jisc1:solrnode6 using 20% of RAM whereas other using less than 5%. No other indicators seen, no recent memory issues reported in Solr logs. Consequently stopped and started jisc1:solrnode6 to clear memory usage. 
The re-read/rebuild of node data takes much time, so we're now in a waiting pattern to see if this resolves the issue.

BTW Due to forthcoming event, I've scheduled to restart all the solr nodes and shine2 this evening. Irrespective of the current problem being resolved, this will go ahead (unless otherwise instructed).

**Comment by GilHoggarth** [comment]
Restarting jisc1:solrnode6 has resolved this outage - /shine is now working again for me.

**Comment by peterwebster** [comment]
Thanks @GilHoggarth ! Hope all is well with you...

**Comment by GilHoggarth** [comment]
Complete restart of all solr nodes and the shine2 service performed successfully last night. It takes several hours, I think about 3, all reported fine. Checking both the Solr service directly and via the /shine URL, all working.
Unless another issue appears or requested otherwise, I'll leave this service now so that it remains stable.
